src/api/java/com/minecolonies/api/util/constant/NameConstants.java

```diff
@@ -6,592 +6,157 @@
-      "Austin",
-      "Benjamin",
-      "Brayden",
-      "Bryson",
-      "Caden",
-      "Caleb",
-      "Callum",
-      "Camden",
-      "Cameron",
-      "Carson",
-      "Carter",
-      "Charles",
-      "Charlie",
-      "Chase",
-      "Christian",
-      "Christopher",
-      "Cole",
-      "Colton",
-      "Connor",
-      "Cooper",
-      "Curtis",
-      "Cyrille",
-      "Damian",
-      "Daniel",
-      "David",
-      "Declan",
-      "Diego",
-      "Diogo",
-      "Dominic",
-      "Duarte",
-      "Dylan",
-      "Easton",
-      "Eli",
-      "Elias",
-      "Elijah",
-      "Elliot",
-      "Ethan",
-      "Evan",
-      "Ezra",
-      "Félix",
-      "Gabriel",
-      "Gavin",
-      "George",
-      "Grayson",
-      "Guewen",
-      "Harrison",
-      "Henrik",
-      "Henry",
-      "Houston",
-      "Hudson",
-      "Hugo",
-      "Hunter",
-      "Ian",
-      "Isaac",
-      "Isaiah",
-      "Jack",
-      "Jackson",
-      "Jacob",
-      "James",
-      "Jason",
-      "Jayce",
-      "Jayden",
-      "Jeremiah",
-      "Jim",
-      "Joel",
-      "John",
-      "Jonathan",
-      "Jordan",
-      "Joseph",
-      "Joshua",
-      "Josiah",
-      "Julian",
-      "Kai",
-      "Karsen",
-      "Kevin",
-      "Kian",
-      "Landon",
-      "Leo",
-      "Levi",
-      "Liam",
-      "Lincoln",
-      "Logan",
-      "Luís",
-      "Lucas",
-      "Luke",
-      "Mark",
-      "Mason",
-      "Mateo",
-      "Matthew",
-      "Max",
-      "Michael",
-      "Miles",
-      "Muhammad",
-      "Nathan",
-      "Nathanael",
-      "Nicholas",
-      "Noah",
-      "Nolan",
-      "Oliver",
-      "Oscar",
-      "Owen",
-      "Parker",
-      "Paul",
-      "Peter",
-      "Philibert",
-      "Rénald",
-      "Ray",
-      "Richard",
-      "Robert",
-      "Rory",
-      "Roxan",
-      "Ryan",
-      "Samuel",
-      "Sebastian",
-      "Steven",
-      "Thaddee",
-      "Thomas",
-      "Tiago",
-      "Tristan",
-      "Tyler",
-      "William",
-      "Wyatt",
-      "Xavier",
-      "Zachary",
-      "Zane",
-      "Abraham",
-      "Allen",
-      "Ambrose",
-      "Arthur",
-      "Avery",
-      "Barnaby",
-      "Bartholomew",
-      "Benedict",
-      "Bernard",
-      "Cuthbert",
-      "Edmund",
-      "Edward",
-      "Francis",
-      "Fulke",
-      "Geoffrey",
-      "Gerard",
-      "Gilbert",
-      "Giles",
-      "Gregory",
-      "Hugh",
-      "Humphrey",
-      "Jerome",
-      "Lancelot",
-      "Lawrence",
-      "Leonard",
-      "Martin",
-      "Mathias",
-      "Nathaniel",
-      "Oswyn",
-      "Philip",
-      "Piers",
-      "Ralph",
-      "Reynold",
-      "Roger",
-      "Rowland",
-      "Simon",
-      "Solomon",
-      "Stephen",
-      "Tobias",
-      "Walter",
-      "William",
-      "Kysen"
+      "Aarav", "Aaron", "Aayan", "Abdiel", "Alain", "Aaliyah", "Abdullah", "Abel", "Abraham", "Abram", "Ace", "Adam", "Adan", "Aden",  "Adonis", "Adrian", "Adriel", "Adrien", 
```

Comment:
you broke the formatting! =P

Reply:
If it doesn't need to be on a new line I'm not making a new line for _over a thousand_ names

Reply:
blasphemy

Reply:
also it's automatic if you use an IDE not GITHUB

Reply:
Okay, give me a few hours to set up an ide 🙃

Reply:
I installed IntelliJ and pulled up the changed file, how do I reformat it?

Reply:
@Asherslab

Reply:
i was joking tbh